tests/checks/interceptor_websocket/interceptor_websocket_test.go

```diff
@@ -184,7 +184,8 @@ spec:
     spec:
       containers:
       - name: websocat-test
-        image: ghcr.io/vi/websocat:v1.14.0
+        # v1.14.0 was amd64-only; :latest includes arm64 (https://github.com/vi/websocat/pull/283)
+        image: ghcr.io/vi/websocat:latest
```

Comment:
wasn't this already solved @wozniakjan ?

Reply:
do you think that curl could be an option? it'd be nice actually. Less dependencies is always better